interoperability-layer-openhim/docker-compose-mongo.cluster.yml

```diff
@@ -11,7 +11,7 @@ services:
   mongo-2:
     image: mongo:4.2
     volumes:
-      - 'openhim-mongo2:/data/db'
+      - 'openhim-mongo-2:/data/db'
```

Comment:
```suggestion
      - 'openhim-mongo-02:/data/db'
```

Reply:
done in [c0d70ba](https://github.com/jembi/platform/pull/197/commits/c0d70ba7882b95805a6b144f3bbf99e7b60f6c01)